Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs

```diff
@@ -744,23 +744,25 @@ public static Task ExecuteJsonRecipeSiteSettingAsync<T>(this UITestContext conte
     public static Task EnableTimeShiftingAsync(this UITestContext context) => context.EnableFeatureDirectlyAsync(ShiftTime);
 
     /// <summary>
-    /// Sets the time shift to a specific value. If both <paramref name="days"/> and <paramref name="seconds"/> are
-    /// provided, then the <see cref="TimeSpan"/> values are added together.
+    /// Sets the time shift to a specific value.
     /// </summary>
-    public static Task SetShiftTimeAsync(this UITestContext context, double days = 0, double seconds = 0)
+    public static Task SetShiftTimeAsync(this UITestContext context, TimeSpan time)
```

Comment:
Here we set/add the time in `Shift` so it still seems appropriate.

Reply:
OK then.